lib/services/project-changes-service.ts

```diff
@@ -293,29 +265,5 @@ export class ProjectChangesService implements IProjectChangesService {
 
 		return changed;
 	}
-
-	private fileChangeRequiresBuild(file: string, projectData: IProjectData) {
-		if (path.basename(file) === PACKAGE_JSON_FILE_NAME) {
```

Comment:
TODO: We need to consider how to handle the case when `package.json` file is changed.